### DISCUSSION TITLE
What would the best approach be, to add one collection data to tracking?

### DISCUSSION THREAD

**Discussion by Anatoly03** [pocketbase/pocketbase#5868] [Q&A]
I would like to add one collection to git, and treat it more like an enum/ list, which should be tracked by the version control manager. An example of such a collection would be blocked_usernames and some other. What would the best approach for this be?

If not, let's discuss if there could be a new collection type in the future, that shares similarities with the `base` collection, but tracked in git. This collection should not be able to have a relation field pointing to local data. Under the hood, it would be an SQlite table locally, but synced with a formatted JSON file outside of `pb_data` (e.g. `pb_store`). In production, these collections should be frozen, but record fields could be updated.

**Comment by ganigeorgiev** [comment]
I don't exactly understand the use case but if you want to have a specific collection and its records as part of version control then the easiest way would be to create a migration (either with [Go](https://pocketbase.io/docs/go-migrations/) or [JS](https://pocketbase.io/docs/js-migrations/)).

There are no plans to add a new collection type for this type of operations unless there is a more prominent use case.

_Side-note: with <= v0.22.x migrations have access only to the transactional db instance, meaning that only direct Dao operations are supported (aka. there are no validations) but with v0.23.0-rc (it is in the `develop` branch) the Dao is merged with the app instance and model persist validations are executed by default._
